packages/web/e2e/utils.ts

```diff
@@ -48,3 +48,16 @@ export const dragAndDropFile = async (
 }
 
 export const resetAuthState = { storageState: { cookies: [], origins: [] } }
+
+export const openCleanBrowser = async ({ browser }: { browser: Browser }) => {
+  const type = browser.browserType()
+  const newBrowser = await type.launch()
+
+  const newPage = await newBrowser.newPage()
+  const context = await newPage.context()
+  // Clear out auth state
+  context.addInitScript(() => {
+    window.localStorage.clear()
```

Comment:
Confirmed; after clearing the localstorage, the old `page` object is still logged in. Which makes sense since they're separate browser contexts (they just both get initialized with entropy auth)